ha/util/conf_store.py

```diff
@@ -221,3 +221,19 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_node_id(index):
+        """
+        Get machine_id for control-node pod.
+        """
+        node_id = ''
+        try:
+            node_key = Conf.search(index, GconfKeys.NODE_CONST.value,
+                                   GconfKeys.TYPE_CONST.value, GconfKeys.CONTROL_NODE.value)
+            for key in node_key:
+                node_id = key.split('>')[1]
+            Log.info(f"Fetched control node machine-id: {node_id}")
+            return node_id
```

Comment:
Suggestion: return can be better outside. So that easy to catch up.

Reply:
Done